src/scheduler.py

```diff
@@ -2,7 +2,7 @@
 #
 # SPDX-License-Identifier: LGPL-2.1-or-later
 #
-# Copyright (C) 2021, 2022, 2023 Collabora Limited
+# Copyright (C) 2021-2025 Collabora Limited
```

Comment:
Probably better to do as separate commit or at least we need to update all files being edited (monitor.py, etc)

Reply:
Yes, I agree. Thanks.